charts/atlantis/stages/production-terraform-cloud/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Use this values file after the initial GitHub app registration. This file needs the ATLANTIS_GH_APP_ID generated by the initial setup process.
+# See initial_setup_values.yaml for the initial setup.
+
+serviceAccount:
+  name: atlantis
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::749942219478:role/atlantis-ops-role
+
+environmentSecrets:
+  - name: ATLANTIS_GH_WEBHOOK_SECRET
+    secretKeyRef:
+      name: atlantis-secret
+      key: github_secret
+
+environment:
+  # Set the ATLANTIS_GH_APP_ID to the id generated by the initial app setup process.
+  ATLANTIS_GH_APP_ID: 114426
+  ATLANTIS_GH_APP_KEY_FILE: /atlantis_keys/gh_app_key_file.pem
+  ATLANTIS_WRITE_GIT_CREDS: true
+  ATLANTIS_ATLANTIS_URL: http://atlantis-terraform-cloud.production.procoretech.com/
+  ATLANTIS_SILENCE_ALLOWLIST_ERRORS: true
+
+github: null
+
+terraformCloud:
+  enable: true
+  orgWhitelist: github.com/procore/atlantis-test-repository
+
+nameOverride: atlantis-terraform-cloud
+
+fullnameOverride: atlantis-terraform-cloud
```

Comment:
Using name overrides in Helm to prevent the StatefulSet names from potentially overwriting

Reply:
this should be fine